api/src/main/java/org/loomdev/api/event/server/TickEvent.java

```diff
@@ -0,0 +1,22 @@
+package org.loomdev.api.event.server;
+
+import org.jetbrains.annotations.NotNull;
+import org.loomdev.api.event.Event;
+import org.loomdev.api.world.World;
+
+public interface TickEvent extends Event {
+
+    long getTick();
+
+    interface Pre extends TickEvent {
+    }
+
+    interface Post extends TickEvent {
+    }
+
+    interface World extends TickEvent {
+
+        @NotNull
+        org.loomdev.api.world.World getWorld();
```

Comment:
Seems like it is. Since the interface is also named "World," the type would be the wrong world unless I put the full path. Just adding the import for World doesn't work.

Reply:
Ok, totally missed that 😓 smh